### PR TITLE
it.js

### PR DESCRIPTION
**Pull request by rqyz**
my discord name is rqyz#8619 and my in game name is giggin07056

### CODE REVIEW

**Comment by mty22** [comment]
Pending review.

**Comment by mty22** [comment]
@MrGira are you able to review this one by any chance?

**Comment by MrGira** [comment]
@mty22 I'm on it

File: it.js [hunk 1]
```diff
@@ -1,863 +1,863 @@
...
+social.market.searching Cercando
+social.market.listed Aggiunto {0} fa
+social.market.list Metti in vendita
+social.market.unlist Rimuovi
+social.market.unlisting Rimuovendo oggetto...
+social.market.posting Mettendo in vendita...
+social.market.req richiesto lvl {0}
+social.market.cant Non puoi permettertelo
+social.market.purchase Acquista
+social.market.fee Gratis!
```
Comment (MrGira) [inline it.js:757]: Personally I would drop the exclamation mark, but I would say the translation works.

File: it.js [hunk 2]
```diff
@@ -1,863 +1,863 @@
...
+windows.host.default.commands *I comandi dell'Host sono disabilitati in questa modalità: /noclip /god /smite etc
+windows.host.rewards Abilitare i premi in KR obbliga il Server all'utilizzo delle impostazioni predefinite e permette ai giocatori di guadagnare KR dai fondi della mappa:
+windows.store.wheels.sub Tenta la tua fortuna, gira la ruota per la possibilità di vincere nuove skin!
+windows.store.currency * I prezzi in USD possono essere convertiti nella tua valuta locale con il tasso di cambio definito dalla tua banca o provider di pagamento
+windows.store.creator Supporta il tuo content creator preferito utilizzando il suo Codice Creatore
+windows.chall.customs Le sfide non possono essere completate nelle partite personalizzate
+twitch.link Sei sicuro di voler aggiungere un account di twitch?
+twitch.unlink Sei sicuro di voler rimuovere il tuo account di twitch?
+windows.clan.leave Sei sicuro di voler lasciare il clan?
+windows.clan.delete Sei sicuro di voler eliminare definitivamente il clan?
```
Comment (MrGira) [inline it.js:823]: Clans can be re-created so I would leave the first translation.

**Review by MrGira — CHANGES_REQUESTED** [review]
The translations is alright, but there are some changes to be done first.
Please review them carefully.

**Comment by mty22** [comment]
Hello @rqyz - can you please re-base and implement suggestions?

**Comment by mty22** [comment]
@rqyz I am going to treat this as abandoned as there's been no response for a while and another PR for `it.js` is active.

Feel free to team up with @MrGira

**Pull request closed without merging**